FFTGeneratorConfigs.scala

```diff
@@ -0,0 +1,23 @@
+package fftgenerator
+
+import freechips.rocketchip.config.{Field, Parameters, Config}
+
+//cparameter to enable FFT
+case object FFTEnableKey extends Field[Option[FixedTailParams]](None)
+
+class WithFFTGenerator (enable: Boolean = true) extends Config((site, here, up) => {
+  case FFTEnableKey => Some(FixedTailParams())
+})
+
+// parameter to set FFT mmio registers base address
+case object FFTBaseAddrKey extends Field[Int](0x2000)
```

Comment:
For any default value can you explain why it was chosen.

Reply:
No longer setting default values